spec/models/card_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'super_memo_2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.